boilerplate/openshift/golang-osd-operator-osde2e/README.md

```diff
@@ -0,0 +1,43 @@
+# Conventions for Ginkgo based e2e tests
+
+- [Conventions for Ginkgo based e2e tests](#conventions-for-ginkgo-based-e2e-tests)
+  - [Consuming](#consuming)
+  - [`make` targets and functions.](#make-targets-and-functions)
+    - [E2E Test Harness](#e2e-test-harness)
+      - [Local Testing](#e2e-harness-local-testing)
+
+
+## Consuming
+Currently, this convention is only intended for OSD operators. To adopt this convention, your`boilerplate/update.cfg` should include:
+
+```
+openshift/golang-osd-operator-osde2e
+```
+## `make` targets and functions.
+
+**Note:** Your repository's main `Makefile` needs to be edited to include the
+"nexus makefile include":
+
+```
+include boilerplate/generated-includes.mk
+```
+
+One of the primary purposes of these `make` targets is to allow you to
+standardize your prow and app-sre pipeline configurations using the
+following:
+
+
+
+### E2e Test Harness
+
+| `make` target      | Purpose                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `e2e-harness-generate` | Generate scaffolding for end to end test harness. Creates osde2e/ directory where tests and test runner reside. The harness has access to cloud client and addon passthrough secrets within the test job cluster. Add your operated related ginkgo e2e tests under the `osde2e/tests/<operator-name>_tests.go` file. See [this README](https://github.com/openshift/osde2e-example-test-harness/blob/main/README.md#locally-running-this-example) for more details on test harness. |
```

Comment:
```suggestion
| `e2e-harness-generate` | Generate scaffolding for an end to end test harness. The `osde2e/` directory is created where the tests and test runner reside. The harness has access to cloud client and addon passthrough secrets within the test job cluster. Add your operater related ginkgo e2e tests under the `osde2e/tests/<operator-name>_tests.go` file. See [this README](https://github.com/openshift/osde2e-example-test-harness/blob/main/README.md#locally-running-this-example) for more details on test harness. |
```

This could use a bit of clearing up. What is a "cloud client"?

Reply:
cloud client would be the connection to AWS or GCP's go sdk. That's what gives the test a hook into the cloud provider.